ccd-definition/AuthorisationCaseField/system-update.json

```diff
@@ -180,5 +180,12 @@
     "CaseFieldID": "confidentialRespondents",
     "UserRole": "caseworker-publiclaw-systemupdate",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "documentToBeSent",
+    "UserRole": "caseworker-publiclaw-systemupdate",
+    "CRUD": "CRU"
```

Comment:
I think that system-update user should be able to delete this field as in code there is cleanup: 
SendDocumentController
caseDetails.getData().remove(DOCUMENT_TO_BE_SENT_KEY);